molecule/default/tests/test_create_and_assign_floating_ip.py

```diff
@@ -0,0 +1,84 @@
+import pytest_rpc.helpers as helpers
+import os
+import pytest
+import testinfra.utils.ansible_runner
+from time import sleep
+
+testinfra_hosts = testinfra.utils.ansible_runner.AnsibleRunner(
+    os.environ['MOLECULE_INVENTORY_FILE']).get_hosts('os-infra_hosts')[:1]
+
+utility_container = ("lxc-attach -n $(lxc-ls -1 | grep utility | head -n 1) "
+                     "-- bash -c '. /root/openrc ; ")
+
+random_str = helpers.generate_random_string(6)
+instance_name = "test_instance_{}".format(random_str)
+image_name = 'Cirros-0.3.5'
+gateway_net = 'GATEWAY_NET'
+private_net = 'PRIVATE_NET'
+flavor = 'm1.tiny'
+floating_ip = None
+
+
+@pytest.mark.test_id('a97e1202-796a-11e8-ba13-525400bd8005')
+@pytest.mark.jira('asc-254')
+@pytest.mark.run(order=1)
+def test_create_floating_ip(host):
```

Comment:
Since the floating ip is only used in the `test_assign_floating_ip_to_instance` test, should the creation be in that test rather than its own test case?

Reply:
I think `create floating IP` could also be a test that need to be done, but I am ok with integrated it to `assign_floating_ip_to_instance`.

Reply:
Ah, thanks. That makes sense to have it be an atomic test.